dev/lint-python

```diff
@@ -210,8 +210,8 @@ function black_test {
     local BLACK_STATUS=
 
     # Skip check if black is not installed.
-    $BLACK_BUILD 2> /dev/null
-    if [ $? -ne 0 ]; then
+    $PYTHON_EXECUTABLE -c 'import black' &> /dev/null
```

Comment:
Could you explain why this should be changed ??

The existing code `"$PYTHON_EXECUTABLE -m black"` isn't enough to check for Black installation ??

Reply:
The reason is very simple:

```
> python -m black
 Usage: python -m black [OPTIONS] SRC ...

 One of 'SRC' or 'code' is required.
> echo $?
1
```

```
> python -m black
/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: No module named black
> echo $?
1
```

```
> python -c 'import black'
> echo $?
0
```

```
> python -c 'import this_does_not_exist
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'this_does_not_exist'
> echo $?
1
```